src/bittytax/templates/ct_estimate.html

```diff
@@ -1,9 +1,9 @@
 {% set cgains = tax_report[tax_year]['CapitalGains'] %}
 <h3>Tax Estimate</h3>
 <p>The figures below are only an estimate, they do not take into consideration other gains and losses in the same tax year, always consult with a professional accountant before filing.</p>
-<table width="75%">
+<table width="100%">
     <tr>
-        <td>Taxable gain:</td>
+        <td width="50%">Taxable gain:</td>
```

Comment:
I'm not sure why but the Tax Estimate table no longer lines up with the CGT summary table above?

![Screenshot 2023-11-22 at 12 35 46](https://github.com/BittyTax/BittyTax/assets/47545109/bd3207e6-393d-4289-96c7-db19ff0d4947)

Reply:
thats strange, the code for both of them is the same:

```
<h3>Summary</h3>
<table width="100%">
    <tr>
        <td width="50%">Number of disposals:</td>
        <td align="right">{{cgains.summary['disposals']}}</td>
        <td></td>
    </tr>
```

and
```
<h3>Tax Estimate</h3>
<p>The figures below are only an e.....</p>
<table width="100%">
    <tr>
        <td width="50%">Taxable gain:</td>
        <td align="right">{{cgains.ct_estimate['taxable_gain']|valuefilter}}</td>
        <td></td>
    </tr>
```

Reply:
lets try at 75% instead. 

I think this line:
`Allowable costs (including the purchase price):` is wrapping around to next line and breaking the 50% width boundary